pkg/services/sqlstore/sqlstore_test.go

```diff
@@ -66,44 +68,53 @@ var sqlStoreTestCases = []sqlStoreTest{
 		dbHost:        "[::1]",
 		connStrValues: []string{"host=::1", "port=5432"},
 	},
+	{
+		name:  "Invalid database URL",
+		dbURL: "://invalid.com/",
+		err:   errors.New("parse \"://invalid.com/\": missing protocol scheme"),
+	},
 }
 
 func TestSQLConnectionString(t *testing.T) {
-	Convey("Testing SQL Connection Strings", t, func() {
-		t.Helper()
+	for _, testCase := range sqlStoreTestCases {
+		t.Run(testCase.name, func(t *testing.T) {
+			sqlstore := &SQLStore{}
+			sqlstore.Cfg = makeSQLStoreTestConfig(t, testCase.dbType, testCase.dbHost, testCase.dbURL)
+			if err := sqlstore.readConfig(); err != nil {
```

Comment:
There's still the problem that if `testCase.err != nil`, it doesn't get caught when `err == nil`.